### ISSUE TITLE
Invalid HTML: could not find <table>

### ISSUE THREAD

**Issue by sinjs** [marlon360/rki-covid-api#245]
When trying to access `/germany`, the following error happens:

```json
{
  "error": {
    "message": "An error occurred.",
    "details": "Invalid HTML: could not find <table>",
    "stack": "Error: Invalid HTML: could not find <table>\n    at html_to_sheet (/usr/src/app/node_modules/xlsx/xlsx.js:19474:19)\n    at Object.html_to_book [as to_workbook] (/usr/src/app/node_modules/xlsx/xlsx.js:19527:28)\n    at parse_xlml_xml (/usr/src/app/node_modules/xlsx/xlsx.js:15826:26)\n    at parse_xlml (/usr/src/app/node_modules/xlsx/xlsx.js:16538:53)\n    at Object.readSync [as read] (/usr/src/app/node_modules/xlsx/xlsx.js:21019:21)\n    at parseRValue (/usr/src/app/src/data-requests/r-value.ts:6:23)\n    at Object.<anonymous> (/usr/src/app/src/data-requests/r-value.ts:40:17)\n    at Generator.next (<anonymous>)\n    at fulfilled (/usr/src/app/dist/data-requests/r-value.js:5:58)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)"
  }
}
```

**Comment by HappyFr1tz** [comment]
The same problem as #244

**Comment by Rubber1Duck** [comment]
and also #243

**Comment by marlon360** [comment]
Fixed with #247
